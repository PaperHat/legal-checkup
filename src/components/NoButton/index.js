import React, { Fragment } from 'react'

import StyledNoButton from '../styled/Button/NoButton'

export default function NoButton () {
  return (
    <Fragment>
      <StyledNoButton>No</StyledNoButton>
    </Fragment>
  )
}
