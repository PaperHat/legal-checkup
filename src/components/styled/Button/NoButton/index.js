import Button from '..'

const NoButton = Button.extend`
  border-top-right-radius: 5px;
  border-bottom-right-radius: 5px;
`

NoButton.displayName = 'NoButton'

export default NoButton
