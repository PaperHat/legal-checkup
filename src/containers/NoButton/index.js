import NoButton from '../../components/NoButton'
import { connect } from 'react-redux'
import { noButtonClicked } from '../../state/actions'

function mapDispatchToProps (dispatch) {
  return {
    handleClick: () => dispatch(noButtonClicked())
  }
}

const ConnectedNoButton = connect(null, mapDispatchToProps)(NoButton)
ConnectedNoButton.displayName = 'ConnectedNoButton'
export default ConnectedNoButton
