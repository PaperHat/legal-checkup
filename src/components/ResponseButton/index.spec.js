import React from 'react'
import { shallow } from 'enzyme'

import NoButton from '.'

describe('components:NoButton', function () {
  it('renders the NoButton and CSS properties properly', () =>
    expect(toJson(shallow(<NoButton>AnswerBar</NoButton>))).toMatchSnapshot())
})
