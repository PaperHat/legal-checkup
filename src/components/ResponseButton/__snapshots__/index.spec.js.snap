// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:NoButton renders the NoButton and CSS properties properly 1`] = `
<React.Fragment>
  <NoButton>
    No
  </NoButton>
</React.Fragment>
`;
